src/representative.jl

```diff
@@ -9,17 +18,106 @@ used for each representative period. In addition, each representative period
 has an associated share that specifies how much of the total duration that
 is attributed to it.
 
+!!! note
+    - All representative periods must use the same type for the `TimeStructure`.
+    - If the field `period_share` is not specified, it assigns the same probability to each
+      representative period.
+    - It is possible that `sum(period_share)` is larger or smaller than 1. This can lead to
+      problems in your application. Hence, it is advised to scale it. We provide currently a
+      warning as it would correspond to a breaking change.
+    - If you include [`OperationalScenarios`](@ref) in your time structure, it is important
+      that you the scenarios are within the representative periods, and note the other way.
```

Comment:
```suggestion
      that you the scenarios are within the representative periods, and not the other way.
```

Reply:
...that the scenarios are within the representative periods, and not the other way around.